deny.toml

```diff
@@ -0,0 +1,17 @@
+[advisories]
+ignore = [
```

Comment:
If valid to in this context, please put the `[` open on its own line to follow the code base style.

Reply:
That's not valid in TOML sadly

Reply:
Ah, well, fair enough - we'll resolve this then and approve. Please rebase on `main` so we can merge.